src/test/lambda/wizards/samDeployWizard.test.ts

```diff
@@ -657,7 +603,8 @@ describe('SamDeployWizard', async () => {
             const wizard = new SamDeployWizard(
                 new MockRegionProvider(),
                 new MockSamDeployWizardContext(
-                    async function*() { yield vscode.Uri.file(templatePath) },
+                    // tslint:disable-next-line:space-before-function-paren
```

Comment:
I think we should probably remove the tslint rules that are conflicting, the formatter should do 
"space-before-function-paren" when it is actually relevent, otherwise we will just have more conflicts in the future. Ditto with line length. 

If PRs slip through not formatted, that is fine because it would have been minor (since no one would have caught it), and it will be reformatted the next time that file is touched (likely)

Reply:
Updated lint rules in #688